### PR TITLE
SDIT-1130: 📝 Document interesting queries

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: hmpps-prisoner-search-indexer/UsefulQueries.md [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
+Most of the interesting information as to what is happening in the indexer is written as `customEvents`:
+```kusto
+customEvents
+| where cloud_RoleName == "hmpps-prisoner-search-indexer"
 | order by timestamp desc
 ```
+There are 3 main events (`name` field):
+* `PRISONER_UPDATED` if a prisoner has changed
+* `PRISONER_OPENSEARCH_NO_CHANGE` if the OpenSearch index hash already contains the changes
+* `PRISONER_DATABASE_NO_CHANGE` if the OpenSearch index is different but the database hash is the same.
```
Comment (mikehalmamoj) [inline hmpps-prisoner-search-indexer/UsefulQueries.md:13]: I'm finding this hard to follow... Is there a difference between the "Opensearch index hash" and the "database hash"? What does it mean if the "OpenSearch index is different but the database hash is the same"?

**Review by mikehalmamoj — APPROVED** [review]

File: hmpps-prisoner-search-indexer/UsefulQueries.md (hunk 1 above)
Comment (petergphillips) [reply hmpps-prisoner-search-indexer/UsefulQueries.md:13]: fair point - will improve in next PR and try to explain in slack
